lib/outcome_checker.rb

```diff
@@ -11,6 +11,10 @@ def draw?
     full_board?(board.combinations) && !win?
   end
 
+  def in_progress?
```

Comment:
I am not a big fan of this method as it is a negation over two negations:
```ruby
def game_is_over
  !outcome_checker.in_progress?
end
```

```ruby
def in_progress?
  !win? && !draw?
end
```

I think I prefer the previous one:
```ruby
def game_is_over
  outcome_checker.win? || outcome_checker.draw?
end
```